resources/functionMap.php

```diff
@@ -8828,8 +8828,8 @@
 'preg_filter' => ['string|array|null', 'regex'=>'string|array', 'replace'=>'string|array', 'subject'=>'string|array', 'limit='=>'int', '&w_count='=>'int'],
 'preg_grep' => ['array|false', 'regex'=>'string', 'input'=>'array', 'flags='=>'int'],
 'preg_last_error' => ['int'],
-'preg_match' => ['int|false', 'pattern'=>'string', 'subject'=>'string', '&w_subpatterns='=>'string[]', 'flags='=>'int', 'offset='=>'int'],
-'preg_match_all' => ['int|false|null', 'pattern'=>'string', 'subject'=>'string', '&w_subpatterns='=>'array', 'flags='=>'int', 'offset='=>'int'],
```

Comment:
per https://www.php.net/preg_match_all `null` cannot be returned

Reply:
Someone somewhere persuaded me it can return null: https://github.com/phpstan/phpstan-src/commit/0d025e7bc2ec6e84a441b2da7813ff933b5a00aa

I'll merge this if you don't remove the `null` type.

Reply:
add back the `null` type

Reply:
In https://3v4l.org/DDBf3, I managed to make it return `null` on PHP 4.3